core/templates/components/question-directives/question-player/question-player.component.ts

```diff
@@ -366,7 +366,7 @@ angular.module('oppia').component('questionPlayer', {
 
           // Calculate scores per skill.
           if (!(questionData.linkedSkillIds)) {
-            continue;
+            return;
```

Comment:
I'm not sure about the changes here so I would like @aks681 to take a pass on these changes before I can approve. 
/cc @aks681

Reply:
The continue function was showing error when I changed to forEach and searched for it and found out return as a solution although even I didnt feel good about changing continue to return @kevintab95

Reply:
I think `return` serves the same purpose as `continue` inside a `forEach` afaik. So the current iteration is stopped there and the rest of the iteration proceeds normally.
Though, @khareyash05 have you verified this? Maybe in a sample array elsewhere?

Reply:
Ya I verified this @aks681